core/new-gui/src/app/dashboard/user/component/user-workflow/user-workflow-list-item/user-workflow-list-item.component.html

```diff
@@ -113,7 +111,7 @@
     *ngIf="userProjectsMap.size > 0"
     class="project-label-container">
     <div
-      *ngFor="let projectID of entry.workflow.projectIDs | unique"
```

Comment:
I think it's better to keep the frontend check, for a defensive purpose. Shall we keep this pipe?

Reply:
We shouldn't do the check on the frontend. i.e. we shouldn't process data on the frontend, and doing so will hide the errors on the backend.

Reply:
After discussion, we decided to keep this check. Pushed this check to the component.